portfolio/src/main/webapp/script.js

```diff
@@ -12,17 +12,26 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-/**
- * Adds a random greeting to the page.
- */
-function addRandomGreeting() {
-  const greetings =
-      ['Hello world!', '¡Hola Mundo!', '你好，世界！', 'Bonjour le monde!'];
 
-  // Pick a random greeting.
-  const greeting = greetings[Math.floor(Math.random() * greetings.length)];
+function addRandomFact() {
+  const facts =
+      ['I am currently 20 years old!', 'I love to hike, bike, and camp!', 'I am an eagle scout!', 'I was the valedictorian of my high school!', 'I love traveling to other countries! I\'ve been to about 10.', 'I love talking to others and socializing with others!'];
```

Comment:
Very nit picky here, so feel free to ignore; from a code cleanliness perspective, having each array element on a new line makes it easier to read and maintain. 

For example, if you need to change one of them, say "I am currently 20 years old!" to 21, with everything on one line, the diff is hard to read. With each array element on one line, you'll clearly see that you just changed the first element.